pmsv2/directdebit/params.go

```diff
@@ -0,0 +1,55 @@
+package directdebit
+
+type ChannelCode string
+type ChannelProperties map[string]interface{}
+type Type string
+
+const (
```

Comment:
let's also remove the channel codes that are not yet supported by PMSv2 - [reference here](https://xendit.atlassian.net/wiki/spaces/CHARGE/pages/2654535704/Supported+Channels)

Reply:
can we also do this to other payment channels?

Reply:
just to clarify, the ones in grey / Under Development should be removed, right?

Reply:
oh alright. I've readded them

Reply:
I think BCA_ONEKLIK, MTB and BDO are not yet supported

Reply:
removed. 

Based on the reference, I don't see BCA_KLIKPAY either. Is that supported yet?

Reply:
right let's remove

Reply:
removed